packages/epo-react-lib/src/form/SelectListbox/SelectListbox.tsx

```diff
@@ -1,23 +1,36 @@
 import { FunctionComponent, useLayoutEffect, useState } from "react";
 import { useUID } from "react-uid";
 import { useTranslation } from "react-i18next";
-import { onChangeCallback, Option } from "@/types/select-listbox";
+import { ListboxOption } from "@/types/select-listbox";
 import IconComposer from "@/svg/IconComposer";
 import * as Styled from "./styles";
 import { useAccessibleDropdown, useOnOutsideClick } from "./hooks";
 
-interface SelectListboxProps {
-  value: string | string[] | null;
-  options: Option[];
-  onChangeCallback: onChangeCallback;
-  isDisabled?: boolean;
-  placeholder?: string;
-  labelledById?: string;
-  namespace?: string;
-  isMultiselect?: boolean;
-  maxWidth?: string;
-  width?: string;
-}
+type SelectListboxProps<TMultiselect = boolean> = TMultiselect extends true
```

Comment:
reconfigured the types for `SelectListbox` so that a single select always takes and receives a `string` and a multiselect always takes and receives a `string[]`